### PR TITLE
Use getDolGlobalString() getDolGlobalInt()

### PR DESCRIPTION
**Pull request by sonikf**

### CODE REVIEW

File: htdocs/accountancy/admin/account.php [hunk 1]
```diff
@@ -509,7 +509,7 @@
 	if (!empty($arrayfields['aa.import_key']['checked'])) {
 		print '<td class="liste_titre"><input type="text" class="flat width75" name="search_import_key" value="'.$search_import_key.'"></td>';
 	}
-	if ($conf->global->MAIN_FEATURES_LEVEL >= 2) {
+	if (getDolGlobalInt('MAIN_FEATURES_LEVEL >= 2) {
```
Comment (frederic34) [inline htdocs/accountancy/admin/account.php:512]: @sonikf missing ')

Comment (hregis) [reply htdocs/accountancy/admin/account.php:512]: @sonikf @frederic34 

	if (getDolGlobalInt('MAIN_FEATURES_LEVEL') >= 2) {

Comment (sonikf) [reply htdocs/accountancy/admin/account.php]: @frederic34 @hregis  Sorry was offline all fixed.

File: htdocs/accountancy/admin/productaccount.php [hunk 2]
```diff
@@ -85,7 +85,7 @@
 	$accounting_product_mode = 'ACCOUNTANCY_SELL';
 }
 
-$limit = GETPOST('limit', 'int') ?GETPOST('limit', 'int') : (empty($conf->global->ACCOUNTING_LIMIT_LIST_VENTILATION) ? $conf->liste_limit : $conf->global->ACCOUNTING_LIMIT_LIST_VENTILATION);
+$limit = GETPOST('limit', 'int') ?GETPOST('limit', 'int') : $accountingLimit = !getDolGlobalInt('ACCOUNTING_LIMIT_LIST_VENTILATION', $conf->liste_limit);
```
Comment (eldy) [inline htdocs/accountancy/admin/productaccount.php:88]: Why an affectation with = here ?

Comment (sonikf) [reply htdocs/accountancy/admin/productaccount.php]: Pointless! No excuse

File: htdocs/accountancy/admin/productaccount.php [hunk 3]
```diff
@@ -85,7 +85,7 @@
 	$accounting_product_mode = 'ACCOUNTANCY_SELL';
 }
 
-$limit = GETPOST('limit', 'int') ?GETPOST('limit', 'int') : (empty($conf->global->ACCOUNTING_LIMIT_LIST_VENTILATION) ? $conf->liste_limit : $conf->global->ACCOUNTING_LIMIT_LIST_VENTILATION);
+$limit = GETPOST('limit', 'int') ?GETPOST('limit', 'int') : !getDolGlobalInt('ACCOUNTING_LIMIT_LIST_VENTILATION', $conf->liste_limit);
```
Comment (eldy) [inline htdocs/accountancy/admin/productaccount.php:88]: Should be
$limit = GETPOST('limit', 'int') ?GETPOST('limit', 'int') : getDolGlobalInt('ACCOUNTING_LIMIT_LIST_VENTILATION', $conf->liste_limit);

Comment (sonikf) [reply htdocs/accountancy/admin/productaccount.php]: fixed, thank you

**Pull request merged**